challenges/classes.js

```diff
@@ -1,7 +1,53 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMaker {
+  constructor(attributes) {
+    this.length = attributes.length;
+    this.width = attributes.width;
+    this.height = attributes.height;
+  }
+  volume() {
+    return this.length * this.width * this.height;
+  }
+
+  surfaceArea() {
+    return (2 * (this.length * this.width + this.length * this.height + this.width * this.height));
+  }
+}
+
+class CubeMaker extends CuboidMaker{
+    constructor(cubeAttributes) {
+    super(cubeAttributes);
+    }
+
+    volume() {
```

Comment:
you don't need to have this volume() property here, as it does the exact same thing as the volume() on CuboidMaker, and because CubeMaker extends CuboidMaker, CubeMaker instances already have access to CuboidMaker's volume method.

If you wanted to give CubeMaker its own version of the method, that's great, but then give it different logic (such as `return this.length ** 3)